packages/core/package.json

```diff
@@ -18,15 +21,16 @@
     "access": "public"
   },
   "dependencies": {
-    "@emotion/core": "^10.0.0",
+    "@emotion/core": "^10.1.1",
     "@theme-ui/css": "^0.4.0-rc.5",
+    "@theme-ui/parse-props": "^0.4.0-rc.5",
     "deepmerge": "^4.2.2"
   },
   "peerDependencies": {
     "react": "^16.11.0"
```

Comment:
Maybe we should make `v16.14.0` the overall React peer dependency version range because technically

```js
import { jsx } from 'theme-ui/jsx-runtime'

jsx('div', {})
```

will break on React below `v16.14.0`.

But then we would force everyone to upgrade React even though not everyone wants to use the automatic JSX runtime.

Reply:
It'd certainly be great not to get warnings about using React 17 if we do explicitly support it, so sounds good to me!

Reply:
I added React v17 to the peerDependency range in e87faca and kept the base version of the dependency. 😊

Reply:
Bumped peer dep to `^16.14.0 || ^17.0.0` 39d1b6c